tests/System.Buffers.Experimental.Tests/BufferExtensionTests.cs

```diff
@@ -1,7 +1,11 @@
-﻿using System.IO.Pipelines.Testing;
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Buffers;
+using System.IO.Pipelines.Testing;
 using Xunit;
 
-namespace System.Buffers.Tests
+namespace System.IO.Pipelines.Tests
```

Comment:
Should we move this file to tests\System.IO.Pipelines.Tests?

Reply:
The file has a dependency on System.Buffers.Experimental

Reply:
I see. What motivated the namespace change then?